autogen.sh

```diff
@@ -5,5 +5,16 @@ set -eu
 srcdir="$(realpath -m "$0"/..)"
 
 (cd "${srcdir}" && autoreconf -is --warnings obsolete)
+[ -n "${NOCONFIGURE:-}" ] && exit
```

Comment:
This syntax still gives me the creeps.. But it does seem to work with `set -e`. Yay shell..

(and if it does fail, we could invert to `[ -z ] || `)